examples/restaurant_demo/README.md

```diff
@@ -0,0 +1,259 @@
+# Example using the Differential Privacy Library PyDP
+
+## Restaurant Demo
+
+Refer Notebook for demo https://github.com/OpenMined/PyDP/blob/dev/examples/restaurant_demo/restaurant_demo.ipynb
+
+Google's readme: https://github.com/google/differential-privacy/tree/main/examples/java
+
+Imagine a fictional restaurant owner named Alice who would like to share
+business statistics with her visitors. Alice knows when visitors enter the
+restaurant and how much time and money they spend there. To ensure that
+visitors' privacy is preserved, Alice decides to use the PyDP library.
+
+## Installation
+To install PyDP, use the PyPI package manager:
+
+pip install python-dp
+(If you have pip3 separately for Python 3.x, use pip3 install python-dp.)
+
+Navigate to `PyDP/examples/restaurant_demo` folder. Execute `Python restaurant.py`
+
+The output will display Private and Non-Private counts for:
+* Count visits by hour of day
+* Count visits by day of week
+* Sum-up revenue per day of the week
+
+Non Private Count is the raw counts and output
+Private count is anonymized Counts and output using PyDP library
+
+## Count visits by hour of day
+
+In this example Alice wants to share this information with potential clients in
+order to let them know the busiest times in the restaurant. For this, we will
+count how many visitors enter the restaurant at every hour of a particular day.
+For simplicity, assume that a visitor comes to the restaurant at most once a
+day. In other words, a visitor is present at most once in the whole dataset.
+
+Visit data for a single day is stored in the `day_data.csv` file. It includes
+the visitor’s ID, the visit duration (in minutes), and the money spent at the
+restaurant.
+
+
+This triggers the logic of `CountVisitsPerHour`. It reads the daily statistics
+and calculates the number of visitors that entered the restaurant every hour of
+the day. The calculation is done twice.
+
+*   First, `CountVisitsPerHour` computes the raw counts and outputs them to
+    `non_private_counts_per_hour.csv`.
+*   Next, `CountVisitsPerHour` calculates private (anonymized) counts using the
+    PyDP library and prints them to
+    `private_counts_per_hour.csv`.
+
+The image below illustrates the results. The Orange (right) bars represent the
+counts without anonymization while blue (left) bars correspond to the private
+(or *anonymized*) counts. You can see that the private values slightly differ
+from the actual ones but the overall trend is preserved. For example, you can
+clearly see that the restaurant is more busy during lunch and dinner time.
+
+![Daily counts](img/counts_per_hour.png)
+
+Note that Differential Privacy involves adding *random noise* to the actual
+data, so your results will most likely be slightly different.
+
+## Partitions and contributions
+
+We say that the resulting aggregated data is split into *partitions*. The bar
```

Comment:
Pls change the "We" to "you" that is the second person or change the sentence.  for example - the resulting aggregated data is split into *partitions*.  ... 

Pls refer to this review https://github.com/OpenMined/PyDP/pull/307#pullrequestreview-508552678 for more detail

Such as 
    In your local (virtual) env, you can install the TensorFlow team's notebook tools:

      python3 -m pip install -U git+https://github.com/tensorflow/docs

 Then use the nblint linter with the --styles=google flag:

         python3 -m tensorflow_docs.tools.nblint --styles=google your_notebook.ipynb

This doesn't highlight everything, but it's good. Check out this README for more options.